content/gotraining.slide

```diff
@@ -425,6 +429,53 @@ As part of all labs
 - Create a PR following the [[https://github.com/anz-bank/go-training#pre-pr-checklist][PR checklist]]
 
 
+* PR from own fork - initial setup
+
+Issue PRs from `feature` branch of your own fork to `master` on upstream:
+
+- Login to github.com
+- Click `Fork` in the top right corner of [[https://github.com/anz-bank/go-training-slides]]
+- Set up the following remotes (`USERNAME` is your GitHub username):
```

Comment:
between these two steps:

- Clone your fork of the repository:
    `git clone https://github.com/USERNAME/go-training.git`
    `cd go-training`

Reply:
The reasons why I didn't describe it this way is that I asked people to clone this repo (upstream) in preparation for the course. I asked them to work through the readme to ensure their environment is set up as required (`go` and `golangci-lint` installed; `go mod` working).

I didn't want to add the complexity of possibly creating a github account, signing into github and creating a fork first.

Maybe I should have had them just discard their local copy and follow your steps though in the training though....
... yes  after some thought: that would have probably been the best..

Done.

Reply:
ok, my comment was just WRT the two steps there. presumably if they are forking, they have set up their github account. but you're on the ground, so whatever works for you.

But I'm also thinking about when the students use these slides as a reference (which is why I've been trying to keep some bad practices out, even if they simply describe a problem/answer/situation).